charforge-gui/backend/app/api/inference.py

```
@@ -0,0 +1,326 @@
+    if character_id:
+        query = query.filter(InferenceJob.character_id == character_id)
+    
+    jobs = query.order_by(InferenceJob.created_at.desc()).offset(offset).limit(limit).all()
+    
+    result = []
+    for job, character in jobs:
+        output_paths = None
+        if job.output_paths:
+            try:
+                output_paths = json.loads(job.output_paths)
+            except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)